float64[3] pos
float64[3] pos_ref
float64[3] v
float64[3] v_ref
float64[3] att
float64 t
float64 thrust
